MODULE.bazel

```diff
@@ -30,31 +30,19 @@ bazel_dep(name = "rules_go", version = "0.46.0", dev_dependency = True, repo_nam
 bazel_dep(name = "rules_nodejs", version = "5.8.2", dev_dependency = True)
 bazel_dep(name = "stardoc", version = "0.6.2", dev_dependency = True, repo_name = "io_bazel_stardoc")
 
-npm = use_extension(
```

Comment:
What are all these changes from?

Reply:
Just formatting cleanup. My main goal here with these pre-factors is to reduce the diff on https://github.com/aspect-build/rules_ts/pull/593 as that is the big hairy one to review.

Reply:
The big hairy one should now be much easier to review: https://github.com/aspect-build/rules_ts/pull/593